lexers/n/numpy_test.go

```diff
@@ -0,0 +1,20 @@
+package n_test
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/alecthomas/assert"
+	"github.com/alecthomas/chroma"
+	"github.com/alecthomas/chroma/lexers/n"
+)
+
+func TestNumPy_AnalyseText(t *testing.T) {
+	data, err := ioutil.ReadFile("testdata/numpy")
```

Comment:
As the text analysis has a more complicated if condition, can we add additional test cases?